### DISCUSSION TITLE
how to debug POST_INIT_link

### DISCUSSION THREAD

**Discussion by rgevaert** [ltsp/ltsp#302] [Q&A]
Hi,

First, I am very impressed with the work done on LTSP. I have used LTSP many years ago. This changes have made the set up much easier!

I am following the wiki page explaining how to set up guest accounts. I am able to boot the client and it auto logs in. However the homedirectory is not cleaned up.

Ref: https://github.com/ltsp/community/wiki/Guest-session

How can I debug this further? 

Also the example states to start xfce, but I am using lightdm, so that is also an issue that will pop up once I get the script executed.

![image](https://user-images.githubusercontent.com/66934/90957877-c0d79980-e490-11ea-8a4c-14a5cab7e00e.png)

![image](https://user-images.githubusercontent.com/66934/90957906-d8168700-e490-11ea-9aee-a21f6a72afee.png)

**Comment by alkisg** [comment]
If you want to debug "POST_INIT_link" you can just put a debug_shell at its end:

```shell
POST_INIT_link="ln -s /etc/ltsp/guest.desktop /usr/share/xsessions/guest.desktop; debug_shell"
```

This will give you a shell right after the command was executed and you can debug what you want. But I don't think that's what you meant. I think you were asking "how can I implement a guest session, as that community wiki page has some issues?"

In general, community wiki pages are maintained by members of the community, not by LTSP developers. Try @mentioning here with @ the LTSP user that wrote it, and see if he's interested in providing feedback and maintaining it.

If not, then I can provide some help regarding community wiki pages, but only to page maintainers. So you may either become that wiki page maintainer, or you can start a new one, and I can help with remarks on fixing the issues it might have.

**Comment by rgevaert** [comment]
Thank you for your help! I will try it to see how I can further troubleshoot.

I have already made some minor corrections on some wiki pages. Thank you for your tips to mention the page owner. If I can contribute with a page I will certainly do. Currently, I am working with a couple of sysadmins in Ethiopia to set up LTSP in their library. :)
